src/client/pydaos/raw/daos_api.py

```diff
@@ -2259,14 +2259,12 @@ def __init__(self, path):
             'disconnect-pool': self.libdaos.daos_pool_disconnect,
             'evict-client':    self.libdaos.daos_pool_evict,
             'exclude-target':  self.libdaos.daos_pool_tgt_exclude,
-            'extend-pool':     self.libdaos.daos_pool_extend,
             'fetch-obj':       self.libdaos.daos_obj_fetch,
             'generate-oid':    self.libtest.dts_oid_gen,
             'get-cont-attr':   self.libdaos.daos_cont_get_attr,
             'get-pool-attr':   self.libdaos.daos_pool_get_attr,
             'get-layout':      self.libdaos.daos_obj_layout_get,
             'init-event':      self.libdaos.daos_event_init,
-            'kill-server':     self.libdaos.daos_mgmt_svc_rip,
```

Comment:
Since daos_mgmt_svc_rip is being removed, don't we need to remove the "kill" method from the DaosServer class too (Line 2206 in this same file)?

Reply:
Good point, @phender it seems 'kill-server' is needed in class DaosServer(object), and it's not a pool operation that dmg  cmd can handle. We may need to keep daos_mgmt_svc_rip/dc_mgmt_svc_rip untouched.

Reply:
If `kill-server` is removed then there really is no need for the [`DaosServer`](https://github.com/daos-stack/daos/blob/master/src/client/pydaos/raw/daos_api.py#L2196) class.  This is used in https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/test_utils_pool.py#L514, but there is a dmg equivalent: https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/test_utils_pool.py#L521

Reply:
@phender  We can remove this condition in 
https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/test_utils_pool.py#L514
self.control_method.value == self.USE_API

Are there other cases? I found at lease this one in src/tests/ftest/pool/pool_svc.py:96
                    server = DaosServer(self.context, self.server_group, 3)
should it be replace with self.USE_DMG as well?